test/e2e/azure_test.go

```diff
@@ -161,6 +161,16 @@ var _ = Describe("Workload cluster creation", func() {
 					}
 				})
 			})
+
+			Context("Validating time synchronization", func() {
```

Comment:
should we run this on the other two specs as well?

Reply:
> should we run this on the other two specs as well?

I don't think it adds much time, so it's probably worthwhile to add to the other two specs since "healthy time sync" is basically a requirement for sane Kubernetes.

Reply:
doesn't it passing in one test an indication that it should pass in all tests?

Reply:
Probably, but not necessarily. If a different cluster has different creation parameters, it's possible that could break something that affects `chrony`, or create a networking issue that affects time sync. And if there's some "once in a hundred VMs" bug relating to time sync, then it would be good to test every one we create.

Having said that, I agree it could yield diminishing returns. I think it's worth adding to the other specs, but I don't feel strongly about it.